src/ensembl/src/shared/accordion/components/AccordionItemButton.tsx

```diff
@@ -58,6 +57,6 @@ const AccordionItemButtonWrapper: React.SFC<WrapperProps> = (
   </ItemConsumer>
 );
 
-AccordionItemButtonWrapper.displayName = DisplayName.AccordionItemButton;
+AccordionItemButtonWrapper.displayName = 'AccordionItemButton';
```

Comment:
Why not delete it at all? Shouldn't `AccordionItemButton` be inferred from the function name (AccordionItemButton)?

Reply:
That's done